src/pybind/mgr/orchestrator/module.py

```diff
@@ -705,61 +705,22 @@ def _apply_osd(self,
                    unmanaged: Optional[bool] = None,
                    dry_run: bool = False,
                    no_overwrite: bool = False,
-                   inbuf: Optional[str] = None) -> HandleCommandResult:
+                   inbuf: Optional[str] = None  # deprecated. Was deprecated before Quincy
+                   ) -> HandleCommandResult:
         """
-        Create OSD daemon(s) using a drive group spec
+        Create OSD daemon(s) on all available devices
         """
-        # Apply DriveGroupSpecs to create OSDs
-        usage = """
-usage:
-  ceph orch apply osd -i <json_file/yaml_file> [--dry-run]
-  ceph orch apply osd --all-available-devices [--dry-run] [--unmanaged]
-
-Restrictions:
-
-  Mutexes:
-  * -i, --all-available-devices
-  * -i, --unmanaged (this would overwrite the osdspec loaded from a file)
-
-  Parameters:
-
-  * --unmanaged
-     Only works with --all-available-devices.
-
-Description:
-
-  * -i
-    An inbuf object like a file or a json/yaml blob containing a valid OSDSpec
-
-  * --all-available-devices
-    The most simple OSDSpec there is. Takes all as 'available' marked devices
-    and creates standalone OSDs on them.
-
-  * --unmanaged
-    Set a the unmanaged flag for all--available-devices (default is False)
-
-Examples:
-
-   # ceph orch apply osd -i <file.yml|json>
-
-   Applies one or more OSDSpecs found in <file>
-
-   # ceph orch osd apply --all-available-devices --unmanaged=true
-
-   Creates and applies simple OSDSpec with the unmanaged flag set to <true>
-"""
 
         if inbuf and all_available_devices:
-            # mutually exclusive
-            return HandleCommandResult(-errno.EINVAL, stderr=usage)
+            return HandleCommandResult(-errno.EINVAL, '-i infile and --all-available-devices are mutually exclusive')
 
         if not inbuf and not all_available_devices:
             # one parameter must be present
-            return HandleCommandResult(-errno.EINVAL, stderr=usage)
+            return HandleCommandResult(-errno.EINVAL, '--all-available-devices is required')
```

Comment:
Do want to just remove this block? It looks like the case with no inbuf or all-available-devices will be caught at the end of the function when we don't enter either the inbuf or all-available-devices block